demos/question-answering/02.serve-vectorstore.ipynb

```diff
@@ -20,7 +20,7 @@
     "\n",
     "In this Notebook, you will be taking the next step in this journey. You will use MLFlow to log the Chroma DB files as artifacts of an experiment. After logging the artifacts, you will then create an Inference Service that retrieves these artifacts and uses them to provide context to user queries. For this purpose, you'll be using KServe, a Kubernetes-based platform that provides a serverless framework for serving machine learning models at scale.\n",
```

Comment:
"In this notebook, you will be performing the following steps in this journey:
1. Use MLflow...experiment.
2. Create a scalable Kubernetes inference service, KServe, that retrieves these artifacts and uses them to provide context to user queries.
"